tools/downloader/converter.py

```diff
@@ -165,6 +165,12 @@ def main():
 
     output_dir = args.download_dir if args.output_dir is None else args.output_dir
 
+    def get_model_dir(model):
+        model_root = (Path(__file__).resolve().parent / '../../models').resolve()
```

Comment:
This duplicates the logic from `common.py`. I would suggest you add a `MODEL_ROOT` constant in `common.py` and reuse it here.

Reply:
Done